package.json

```diff
@@ -34,6 +34,7 @@
     "lighthouse-logger": "2.0.1",
     "multi-progress-bars": "^5.0.3",
```

Comment:
any chance this could also get rid of the current progress bar? Or is this for next steps?

Reply:
Definitely, yes. But I'll do that in another PR, for now the logger is still being used.